src/azure-cli-testsdk/azure/cli/testsdk/utilities.py

```diff
@@ -54,7 +54,7 @@ def _py3_byte_to_str(byte_or_str):
     logger = logging.getLogger()
     logger.warning(type(byte_or_str))
     try:
-        return str(byte_or_str, 'utf-8') if isinstance(byte_or_str, bytes) else byte_or_str
+        return str(byte_or_str, 'utf-8', errors='ignore') if isinstance(byte_or_str, bytes) else byte_or_str
```

Comment:
Why ignore encoding error here?

Reply:
@haroldrandom This modification intended to bypass the VCR recoding error because in this case the request body is a zip file content, not a valid UTF-8. But this would cause replay process fail, so I decided to revert this change, and just make the test as LiveScenarioTest.

Reply:
LiveScenarioTest is always time-consuming, for furthor development, we should make it possible to record/replay binary requests.